dev-vcs/git-publish/git-publish-1.7.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+inherit python-single-r1
+
+DESCRIPTION="Tool for preparing and storing patch revisions as git tag"
+HOMEPAGE="https://github.com/stefanha/git-publish"
+SRC_URI="https://github.com/stefanha/git-publish/archive/v${PV}.tar.gz -> $P.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="man"
+
+DEPEND=""
```

Comment:
We don't need to [define](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-620007.3) some empty variables, so we can drop it